kasa/tapo/tapodevice.py

```diff
@@ -114,16 +149,6 @@ def rssi(self) -> Optional[int]:
         rssi = self._info.get("rssi")
         return int(rssi) if rssi else None
 
-    @property
-    def mac(self) -> str:
-        """Return the mac formatted with colons."""
-        return str(self._info.get("mac")).replace("-", ":")
-
-    @property
-    def device_id(self) -> str:
```

Comment:
Please avoid reorganizing methods while making changes to implementation, that will keep the diffs more readable.

Reply:
Put back in the original place